app/controllers/sitemap_controller.rb

```diff
@@ -0,0 +1,46 @@
+class SitemapController < ApplicationController
+  DEFAULT_LASTMOD = "2021-03-01".freeze
+
+  ALIASES = {
+    "/home" => "/",
+  }.freeze
+
+  OTHER_PATHS = %w[
+    /events
+    /event-categories/train-to-teach-events
+    /event-categories/online-q-as
+    /event-categories/school-and-university-events
```

Comment:
Should the privacy policy/cookies/guidance pages be in here? and would it be worth indexing the sign up for the mailing list initial page?